config/checkstyle_checks.xml

```diff
@@ -551,6 +551,7 @@
     <module name="JavadocMethod">
       <property name="validateThrows" value="true"/>
     </module>
+    <module name="JavadocMissingLeadingAsterisk"/>
```

Comment:
Google style did not have it, sun style is very undeveloped by us, I am ok to do this as separate commit if there is reference for this.
Javadoc style guide is different document from sun java style.

Reply:
>Do we want to make this a requirement for all new module reviews?

Google style was throughly review ed on moment of initial implementation, and we created all requirements Checks to cover it.

Style is changing, and some new stuff might appear in but it will appear in version that we do not support yet, so betty to do this in separate issue.

Reply:
https://checkstyle.org/styleguides/google-java-style-20180523/javaguide.html#s7.1.2-javadoc-paragraphs
> a line containing only the aligned leading asterisk (*)—appears between paragraphs, and before the group of block tags if present. 

They just specify limited usage. It doesn't say each line has to have one.

Reply:
It is about between paragraph.
I still do not see explicit requirement for leading asterisk to be on each line.
Better to avoid activation of this Check now.

Reply:
Yes, all leading asterisk are optional. This check is about to make comments like [this](https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/issue-6722-missing-asterisk_2020130748/reports/diff/apache-ant/xref/home/runner/work/checkstyle/checkstyle/contribution/checkstyle-tester/repositories/apache-ant/src/main/org/apache/tools/ant/taskdefs/JDBCTask.java.html#L45) more readable.